### DISCUSSION TITLE
CVE-2022-24894 basic auth and csrf protection

### DISCUSSION THREAD

**Discussion by nicolasleborgne** [symfony/symfony#49188] [Q&A] [answered]
Hi,

Am I wrong or since the security fix for CVE-2022-24894 we cannot use in combination http basic auth and csrf protection ? Tokens are cleared at each successful login and login occurred at each request so, when submitting a form, tokens do not match.

Maybe I am misunderstanding something ? (@nicolas-grekas 😇)

Thanks

**Comment by nicolas-grekas** [comment]
Can you provide a reproducer to understand what you mean and have ?

**Comment by nicolas-grekas** [comment, marked as answer]
Let's discuss this in https://github.com/symfony/symfony/issues/49194
